sml/asyncio.py

```diff
@@ -126,10 +131,23 @@ async def connect(self, loop=None):
         await self._reconnect(delay=0)
 
     async def _disconnect(self):
+        if self._watchdog and not self._watchdog.done():
+            self._watchdog.cancel()
         if self._transport:
             self._transport.abort()
             self._transport = None
 
+    async def _timeout(self):
+        timeout_delay = 120
+        self._LAST_UPDATE = time.time() + timeout_delay
+        while True:
+            await asyncio.sleep(30)
```

Comment:
Why 30s and not until self._last_update + timeout_delay?

Reply:
I changed it to the timeout_delay. 30 seconds were my initial delay while testing. `self._last_update + timeout_delay` would be too long i think